pywwt/layers.py

```diff
@@ -236,40 +256,72 @@ class TableLayer(HasTraits):
+    alt_type = Unicode(help='The type of altitude (`str`)').tag(wwt='altType')
 
-    size_scale = Float(10, help='The factor by which to scale the size of the points').tag(wwt='scaleFactor')
+    size_scale = Float(10, help='The factor by which to scale the size '
+                       'of the points (`float`)').tag(wwt='scaleFactor')
 
-    # NOTE: we deliberately don't link size_att to sizeColumn because we need to
-    # compute the sizes ourselves based on the min/max and then use the
+    # NOTE: we deliberately don't link size_att to sizeColumn because we need
+    # to compute the sizes ourselves based on the min/max and then use the
     # resulting column.
-    size_att = Unicode(help='The column to use for the size')
-    size_vmin = Float(None, allow_none=True)
-    size_vmax = Float(None, allow_none=True)
+    size_att = Unicode(help='The column to use for the marker size '
+                       '(`str`)').tag(wwt=None)
+    size_vmin = Float(None, help='The minimum point size. '
+                      'Found automagically once size_att is set '
+                      '(`float`)', allow_none=True).tag(wwt=None)
+    size_vmax = Float(None, help='The maximum point size. '
+                      'Found automagically once size_att is set '
+                      '(`float`)', allow_none=True).tag(wwt=None)
 
     # NOTE: we deliberately don't link cmap_att to colorMapColumn because we
-    # need to compute the colors ourselves based on the min/max and then use the
-    # resulting column.
-    cmap_att = Unicode(help='The column to use for the colormap')
-    cmap_vmin = Float(None, allow_none=True)
-    cmap_vmax = Float(None, allow_none=True)
-    cmap = Any(cm.viridis, help='The Matplotlib colormap')
-
-    color = Color('white', help='The color of the markers').tag(wwt='color')
-    opacity = Float(1, help='The opacity of the markers').tag(wwt='opacity')
-
-    marker_type = Unicode('gaussian', help='The type of marker').tag(wwt='plotType')
-    marker_scale = Unicode('screen', help='Whether the scale is defined in '
-                           'world or pixel coordinates').tag(wwt='markerScale')
-
-    far_side_visible = Bool(False, help='Whether markers on the far side are '
-                            'visible').tag(wwt='showFarSide')
+    # need to compute the colors ourselves based on the min/max and then use
+    # the resulting column.
+    cmap_att = Unicode(help='The column to use for the colormap '
+                       '(`str`)').tag(wwt=None)
+    cmap_vmin = Float(None, help='The minimum level of the colormap. Found '
+                      'automagically once cmap_att is set (`float`)',
+                      allow_none=True).tag(wwt=None)
+    cmap_vmax = Float(None, help='The maximum level of the colormap. Found '
+                      'automagically once cmap_att is set (`float`)',
+                      allow_none=True).tag(wwt=None)
+    cmap = Any(cm.magma, help='The Matplotlib colormap '
```

Comment:
Why change the default? I thought viridis would make sense for consistency with matplotlib

Reply:
This was a test I was running that I didn't return to its original value. I'll restore viridis in the next commit.